atomicapp/nulecule/base.py

```diff
@@ -128,10 +131,27 @@ def load_from_path(cls, src, config=None, namespace=GLOBAL_CONF,
             an image).
         """
         nulecule_path = os.path.join(src, MAIN_FILE)
+
+        if os.path.exists(nulecule_path):
+            nulecule_data = open(nulecule_path, 'r').read()
```

Comment:
should we use the contextlib stuff here:

```
with open() as f:
  nulecule_data = f.read()
```

I think that would be good practice.